plugins/drivers/proto/driver.proto

```diff
@@ -525,10 +525,9 @@ message LinuxResources {
     int64 memory_limit_bytes = 4;
     // OOMScoreAdj adjusts the oom-killer score. Default: 0 (not specified)
     int64 oom_score_adj = 5;
-    // CpusetCpus constrains the allowed set of logical CPUs. Default: "" (not specified)
-    string cpuset_cpus = 6;
-    // CpusetMems constrains the allowed set of memory nodes. Default: "" (not specified)
-    string cpuset_mems = 7;
```

Comment:
I don't think we can safely remove fields we've [already shipped](https://github.com/hashicorp/nomad/blob/v1.0.4/plugins/drivers/proto/driver.proto#L527-L530)... we own the drivers that we ship, but the community drivers will all be broken.

Reply:
These fields are never used, never even set as far as I could tell. I'm ok leaving them in, but the Nomad client never sets a value for them (as far as I can tell).

Reply:
I think we're supposed to "reserve" the existing field numbers rather than reuse them in that case.

Reply:
It looks like this breaks our linter anyways (for good reason) so I'll just revert and make a new field:
```
==> Checking for breaking changes in protos...
plugins/drivers/proto/driver.proto:516:1:Previously present field "7" with name "cpuset_mems" on message "LinuxResources" was deleted without reserving the name "cpuset_mems".
plugins/drivers/proto/driver.proto:516:1:Previously present field "7" with name "cpuset_mems" on message "LinuxResources" was deleted without reserving the number "7".
plugins/drivers/proto/driver.proto:529:5:Field "6" with name "cpuset_cgroup" on message "LinuxResources" changed option "json_name" from "cpusetCpus" to "cpusetCgroup".
plugins/drivers/proto/driver.proto:529:12:Field "6" on message "LinuxResources" changed name from "cpuset_cpus" to "cpuset_cgroup".
make: *** [GNUmakefile:137: check] Error 1
```